src/qt/walletmodel.cpp

```diff
@@ -54,6 +54,20 @@ WalletModel::~WalletModel()
     unsubscribeFromCoreSignals();
 }
 
+void WalletModel::loadWalletOptions()
+{
+    // update optionsModel with wallet options
+    optionsModel->setSST(static_cast<double>(getWalletStakeSplitThreshold() / COIN));
+    optionsModel->refreshDataView();
```

Comment:
`refreshDataView` will refresh the data for every element in the settings. Better to only update the view for the SST row.

Reply:
removed the refresh completely, and added the signal update.